.github/scripts/print_latest_commits.py

```diff
@@ -56,7 +58,21 @@ def get_commit_results(commit: str, results: Dict[str, Any]) -> List[Dict[str, A
             )._asdict())
     return workflow_checks
 
-def isGreen(results: List[Dict[str, Any]]) -> bool:
+def isGreen(commit: str, results: Dict[str, Any]) -> bool:
```

Comment:
we may want to return some type of error reason here in order to say like what jobs are failing